COLLIDER_EVENTS.md

```diff
@@ -0,0 +1,66 @@
+# Overview
+
+Collider events from the client are broken down in to two types: Collision and Trigger.  Each of these event types are raised for one of two event states: 'enter' or 'exit'.
+
+# Collision Event
+
+The collision event is raised when two actors that both have colliders on them participate in a collision event, one for when the collision enters and one for when the collision exits..  This collision event will contains data that is passed to the event handlers attached for the collision event and state.
+
+## Collision Data
+
+Collision data is specific to only collision events and contains information on the other actor, as well as physics related information about the collision.
+
+```ts
+export interface ContactPoint {
```

Comment:
And, maybe later we can add the app the actor came from :)

Reply:
I am intentionally filtering out actors that are not in the same app for the first pass.  We need an actual design on the inter-app communication.  😃